script/tool/test/publish_plugin_command_test.dart

```diff
@@ -49,8 +56,7 @@ void main() {
     processRunner = TestProcessRunner();
     mockStdin = MockStdin();
     commandRunner = CommandRunner<void>('tester', '')
-      ..addCommand(PublishPluginCommand(
-          mockPackagesDir, mockPackagesDir.fileSystem,
+      ..addCommand(PublishPluginCommand(mockPackagesDir, fileSystem,
```

Comment:
I'd like to move away from `mockPackagesDir`, since it's weirdly indirect. `parentDir` is the same directory, but much more clear based on local context.

Reply:
done